src/shaders/metallic-roughness.frag

```diff
@@ -332,7 +323,7 @@ void main()
 #endif
 
 #ifdef MATERIAL_UNLIT
-    gl_FragColor = vec4(LINEARtoSRGB(baseColor.rgb), baseColor.a);
+    exportColor( vec4(LINEARtoSRGB(baseColor.rgb), baseColor.a));
```

Comment:
This doesn't work because the function doesn't exist.
Just assign `g_finalColor` directly

Reply:
you can verify with the UnlitTest asset